files/ceph-version-qemu.sh

```diff
@@ -30,11 +30,15 @@ PIDS=$(pidof /usr/libexec/qemu-kvm)
 
 for PID in $PIDS; do
   QEMU_TIMESTAMP=$(date -d "$(stat -c %x /proc/$PID/stat)" +%s)
-  if [[ "$QEMU_TIMESTAMP" -lt "$CEPH_UPGRADE_TIMESTAMP" ]]
+  INSTANCE_NAME=$(ps -ef -q $PID|grep -Po "(instance-\w+)"|uniq)
+  INSTANCE_QEMU_MON_RBD=$(virsh qemu-monitor-command --hmp "$INSTANCE_NAME" 'info block'|grep rbd)
+  INSTANCE_USE_RBD=$?
+  if [[ "$QEMU_TIMESTAMP" -lt "$CEPH_UPGRADE_TIMESTAMP" ]] && [ "$INSTANCE_USE_RBD" -eq "0" ]
```

Comment:
How does this works if an instance has been booted on the same day as ceph was upgraded?

Reply:
Yum event begin time is the reference point, anything before that is considered to be the old version